### PR TITLE
LPS-116792 - Add new global menu mobile pattern

### PR DESCRIPTION
**Pull request by marcoscv-work**
- This PR adds the new behavior for global menu in mobile, here an example:

global-menu-mobile

- To test the PR you only have to open the global menu in mobile or resize your screen as you can see in the previous gif

### CODE REVIEW

**Comment by marcoscv-work** [comment]
ci:test:sf

**Comment by marcoscv-work** [comment]
ci:test:relevant

**Comment by jbalsas** [comment]
LGTM superficially 👍

File: modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss [hunk 1]
```diff
@@ -5,6 +5,27 @@ $tab-header-height: 38px;
 
 .global-apps-menu-modal {
 	.global-apps-menu-header {
+		.autofit-row {
+			@include media-breakpoint-down(xs) {
```
Comment (eduardoallegrini) [inline modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:9]: I don't love the `media-breakpoint-down` but I guess it was necessary and "cheaper" than `media-breakpoint-up` 😁

**Comment by eduardoallegrini** [comment]
ci:forward

File: modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss (hunk 1 above)
Comment (marcoscv-work) [reply modules/apps/product-navigation/product-navigation-global-menu-web/src/main/resources/META-INF/resources/css/GlobalMenu.scss:9]: yeah, I've tried to do it by default helpers but is not possible, it is a corner case so I think the best is a custom CSS

**Pull request closed without merging**